changes/634.fixed

```diff
@@ -0,0 +1,4 @@
+ServiceNow Integration
```

Comment:
Please rephrase this to be a bit more concise. This is the changelog fragment that will be added to the release notes when we cut the next release. This should be a single line or two at most.

Reply:
I’ve made the change. I hope it’s better now.